packer/reporter.json

```diff
@@ -1,59 +1,75 @@
-            "filters": {
-                "virtualization-type": "hvm",
-                "name": "debian-stretch-hvm-x86_64-gp2-*",
-                "root-device-type": "ebs"
-            },
-            "owners": [
-                "379101102735"
-            ],
-            "most_recent": true
-        },
-        "instance_type": "t3.medium",
-        "ssh_username": "admin",
-        "ami_name": "cyhy-reporter-hvm-{{timestamp}}-x86_64-ebs",
-        "ami_regions": [
-            "us-east-1",
-            "us-west-1",
-            "us-west-2"
-        ],
-
-        "ami_block_device_mappings": [{
-            "device_name": "xvda",
-            "volume_size": 8,
-            "volume_type": "gp2",
-            "encrypted": true,
-            "delete_on_termination": true
-        }],
-
-        "launch_block_device_mappings": [{
-            "device_name": "xvda",
-            "volume_size": 8,
-            "volume_type": "gp2",
-            "encrypted": true,
-            "delete_on_termination": true
-        }],
-
-        "tags": {
-            "Team": "NCATS OIS - Development",
-            "Application": "Cyber Hygiene",
-            "OS_Version": "Debian Stretch",
-            "Release": "Latest",
-            "Base_AMI_Name": "{{ .SourceAMIName }}"
+  "builders": [
+    {
+      "ami_block_device_mappings": [
+        {
+          "delete_on_termination": true,
+          "device_name": "xvda",
+          "encrypted": true,
+          "volume_size": 8,
+          "volume_type": "gp2"
         }
-    }],
-
-    "provisioners": [
+      ],
+      "ami_name": "cyhy-reporter-hvm-{{timestamp}}-x86_64-ebs",
+      "ami_regions": [
+        "us-east-1",
+        "us-west-1",
+        "us-west-2"
+      ],
+      "instance_type": "t3.medium",
+      "launch_block_device_mappings": [
         {
-            "type": "ansible",
-            "playbook_file": "packer/ansible/playbook.yml",
-            "groups": [
-                "cyhy_reporter"
-            ]
+          "delete_on_termination": true,
+          "device_name": "xvda",
+          "encrypted": true,
+          "volume_size": 8,
+          "volume_type": "gp2"
         }
-    ]
+      ],
+      "region": "us-east-2",
+      "source_ami_filter": {
+        "filters": {
+          "name": "debian-stretch-hvm-x86_64-gp2-*",
+          "root-device-type": "ebs",
+          "virtualization-type": "hvm"
+        },
+        "most_recent": true,
+        "owners": [
+          "379101102735"
+        ]
+      },
+      "ssh_username": "admin",
+      "tags": {
+        "Application": "Cyber Hygiene",
+        "Base_AMI_Name": "{{ .SourceAMIName }}",
+        "OS_Version": "Debian Stretch",
+        "Release": "Latest",
+        "Team": "NCATS OIS - Development"
+      },
+      "type": "amazon-ebs"
+    }
+  ],
+  "provisioners": [
+    {
+      "groups": [
+        "reporter"
+      ],
+      "playbook_file": "packer/ansible/upgrade.yml",
+      "type": "ansible"
+    },
+    {
```

Comment:
The python playbook removes python2 from the AMI.  Are the reporter, nmap, nessus, mongo, docker, and dashboard AMIs all ready for such a big step?

Reply:
The Python playbook has `remove_python2 separated into a separate task (I didn't just blindly copy it). That task only applies to the `bastion` host at the moment.

Reply:
Instance types will be added as they become Python 3 only, and eventually it will just be one task when they are all on Python 3 only.

Reply:
Ah, I missed that.  Thanks for pointing it out.